tools/scripts/build.ps1

```diff
@@ -1,14 +1,32 @@
+.\gradlew --stop
```

Comment:
Rather than do this, it would be better to not run the gradle daemon in the first place. The easiest way to accomplish this is to add `--no-daemon` to the gradlew run in non-Appveyor mode (it already is added in Appveyor mode).

Reply:
I'll do that, thanks.